### PR TITLE
Updated Guest OS support matrix as per CP-6138 and DP-2962

### PR DESCRIPTION
**Pull request by Mona2012**
The Guest OS support matrix is updated in the document as per this sheet - 
https://docs.google.com/spreadsheets/d/1TRXV1A0BI4nr8gtKSHzYZJDX2vDz3QkisCWG6Az1zQY/edit#gid=0

### CODE REVIEW

**Review by ayyajpio — APPROVED** [review]
Looks good to me.

File: pages/04.HDM Doc 2.2/18.appendix/default.md [hunk 1]
```diff
@@ -170,54 +170,95 @@ The document describes the interactions between HDM components and VMware Infras
...
+| RHEL          | RHEL 8.3                        | UEFI Secure | Not Supported          | Not Supported          | Qualified           | Qualified                    | Not Supported                |
+| SLES          | SLES 12-SP5                     | MBR         | Qualified              | Qualified              | Qualified           | Qualified                    | Qualified                    |
+| SLES          | SLES 12-SP5                     | UEFI        | Qualified              | Not qualified          | Qualified           | Qualified                    | Qualified                    |
+| SLES          | SLES 12-SP5                     | UEFI Secure | Not Supported          | Not Supported          | Qualified           | Qualified                    | Not Supported                |
+| SLES          | SLES 15-SP2                     | MBR         | Qualified              | Not qualified<br>      | Qualified           | Qualified                    | Qualified                    |
```
Comment (kgcb) [inline pages/04.HDM Doc 2.2/18.appendix/default.md]: Additional _br_ required here?

File: pages/04.HDM Doc 2.2/18.appendix/default.md [hunk 2]
```diff
@@ -170,54 +170,95 @@ The document describes the interactions between HDM components and VMware Infras
...
+| Ubuntu        | Ubuntu Server 16.04             | MBR         | Qualified              | Qualified              | Qualified           | Qualified                    | Qualified                    |
+| Ubuntu        | Ubuntu Server 16.04             | UEFI        | Not Supported          | Not Supported          | Qualified           | Qualified                    | Not Supported                |
+| Ubuntu        | Ubuntu Server 16.04             | UEFI Secure | Not Supported          | Not Supported          | Qualified           | not qualified(CP-6117)       | Not Supported                |
+| Ubuntu        | Ubuntu Server 18.04             | MBR         | Qualified              | Qualified              | Qualified           | Qualified                    | Qualified                    |
+| Ubuntu        | Ubuntu Server 18.04             | UEFI        | not qualified(CP-5956) | Qualified              | Qualified           | Qualified                    | not qualified(CP-5956)       |
```
Comment (kgcb) [inline pages/04.HDM Doc 2.2/18.appendix/default.md]: _not -> Not_

File: pages/04.HDM Doc 2.2/18.appendix/default.md [hunk 3]
```diff
@@ -170,54 +170,95 @@ The document describes the interactions between HDM components and VMware Infras
...
+| Ubuntu        | Ubuntu Server 18.04             | UEFI Secure | Not Supported          | Not Supported          | Qualified           | Qualified                    | Not Supported                |
+| Ubuntu        | Ubuntu Server 20.04             | MBR         | Qualified              | Qualified              | Qualified           | Qualified (Need SI handling) | Qualified                    |
+| Ubuntu        | Ubuntu Server 20.04             | UEFI        | Qualified              | Qualified              | Qualified           | Qualified (Need SI handling) | Qualified                    |
+| Ubuntu        | Ubuntu Server 20.04             | UEFI Secure | Not Supported          | Not Supported          | Qualified           | Qualified (Need SI handling) | Not Supported                |
+| Oracle\_Linux | Oracle\_Linux 6.8               | MBR         | not qualified(DP-2962) | not qualified(DP-2962) | Qualified           | Qualified                    | not qualified(DP-2962)       |
```
Comment (kgcb) [inline pages/04.HDM Doc 2.2/18.appendix/default.md]: _not -> Not_
Also keep the **(** consistent. In the rows above this there is space before (. But here it is not.

File: pages/04.HDM Doc 2.2/18.appendix/default.md (hunk 2 above)
Comment (Mona2012) [reply pages/04.HDM Doc 2.2/18.appendix/default.md]: Resolved..

File: pages/04.HDM Doc 2.2/18.appendix/default.md (hunk 3 above)
Comment (Mona2012) [reply pages/04.HDM Doc 2.2/18.appendix/default.md]: Resolved

**Pull request merged**